### ISSUE TITLE
Help Please: Print Preview Failed

### ISSUE THREAD

**Issue by robrez** [PolymerElements/paper-drawer-panel#84]
Printing fails, "Print Preview Failed", if you attempt to print a page that uses a paper-drawer-panel.

```
<body class="fullbleed relative">
  <paper-drawer-panel class="fit">
    <div drawer class="content">Nav</div>
    <div main>Oh Hello</div>
  </paper-drawer-panel>
</body>
```

You can see an example of this over at the [element catalog](https://elements.polymer-project.org/browse?package=paper-elements)

I'm not quite sure how much of this may relate to shadow/light-dom versus the paper-drawer-panel specifically. Any suggestions are be appreciated.

I've noticed the [polymer-project devguide](https://www.polymer-project.org/1.0/docs/devguide/feature-overview.html) doesn't share this problem. It uses a different custom element, `app-drawer` in conjunction with a `scroll-area` element whose padding-left changes based on the drawer's state. I may borrow an approach similar to that...

I started down a nasty path of adding a "Print" action button from which I clone the content inside the main portion of the drawer-panel into an iframe and print via it.... eww :-1:

**Comment by robrez** [comment]
_update_

I updated polymer, the trivial code gist/snippet I gave in my previous comment no longer fails.

I've since been trying to create a small example that duplicates the issue and I haven't been able to make much sense out of things.

I do think that the "problem" isn't necessarily related specifically to paper-drawer-panel; instead, I think it has something to do w/ nesting certain elements and/or layout attributes beyond some tipping point that I can't quite figure out yet (in terms of a trivial example).
